src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/InlineCompletion/InlineCompletionEndPoint.cs

```diff
@@ -148,7 +152,7 @@ public TextDocumentIdentifier GetTextDocumentIdentifier(VSInternalInlineCompleti
         _logger.LogInformation($"Returning {items.Count} items.");
         return new VSInternalInlineCompletionList
         {
-            Items = items.ToArray()
+            Items = [.. items]
```

Comment:
For the most part in this PR I didn't look too deeply into the guts of what each class did, and even less so into things that weren't inside the formatting layer. The changes in `RazorFormattingPass` to use a pooled array builder, and the frozen set change, were in reaction to an item in the error list. I'll have a look at this, but might leave some of these sorts of things to a follow up, because I feel like this PR is big enough already. For example, I have a note to myself to change the formatting engine over to TextChange from TextEdit, as I suspect that would be a huge allocation win, but out of scope

Reply:
Okay, this change was simple and had no flow on effects, so I've updated